src/Compilers/CSharp/Portable/Compilation/MethodBodySemanticModel.cs

```diff
@@ -9,6 +8,27 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal sealed class MethodBodySemanticModel : MemberSemanticModel
     {
+#nullable enable
+        /// <summary>
+        /// Initial state for a MethodBodySemanticModel. Shared between here and the <see cref="MethodCompiler"/>.
```

Comment:
>Initial state for a MethodBodySemanticModel. Shared between here and the <see cref="MethodCompiler"/>. [](start = 12, length = 102)

nit: Consider "Used to make a MethodBodySemanticModel and we already have some/all the parts we'll need" or something along those lines